kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -484,6 +498,13 @@ public static ConfigDef getConfig() {
             MERGE_RECORDS_THRESHOLD_VALIDATOR,
             MERGE_RECORDS_THRESHOLD_IMPORTANCE,
             MERGE_RECORDS_THRESHOLD_DOC
+        ).define(
+            TIME_PARTITIONING_TYPE_CONFIG,
+            TIME_PARTITIONING_TYPE_TYPE,
+            TIME_PARTITIONING_TYPE_DEFAULT,
+            ConfigDef.CaseInsensitiveValidString.in(TIME_PARTITIONING_TYPES.toArray(new String[0])),
+            TIME_PARTITIONING_TYPE_IMPORTANCE,
```

Comment:
Should we include a recommender for the enum?

Reply:
+1

Reply:
👍  done